spec/acts_as_taggable_on/taggable_spec.rb

```diff
@@ -161,6 +161,25 @@
     expect(@taggable.skill_list.sort).to eq(%w(ruby rails css).sort)
   end
 
+  context 'when invalid tag name is passed' do
+    context 'and call save' do
+      it 'should not be able to create tags' do
+        @taggable.tag_list << ('a' * 256)
+
+        expect{ @taggable.save }.not_to change(ActsAsTaggableOn::Tag, :count)
+        expect{ @taggable.save }.not_to raise_error
+      end
+    end
+
+    context 'and call save!' do
+      it 'should raise validation error' do
+        @taggable.tag_list << ('a' * 256)
+
+        expect{ @taggable.save! }.to raise_error(ActiveRecord::RecordInvalid, /Validation failed: Name is too long/)
```

Comment:
Before patch applied, receive `ActiveRecord::RecordInvalid: Validation failed: Tag can't be blank` on here.